char_gen.go

```diff
@@ -87,6 +87,24 @@ var charTypeNamesByFlag = map[CTFlag]string{
 	All:     "All characters",
 }
 
+// Re-generation trials for meeting requirements
+var (
+	MaxTrials   = 200              // MaxTrials how many time we try to generate before giving up
+	MinFailRate = 1.0 / 1000000000 // Minimum acceptable failure rate after MaxTrials
```

Comment:
This should be MaxFailRate

```suggestion
	MaxFailRate = 1.0 / 1000000000 // Maximum acceptable failure rate after MaxTrials
```